known_names.py

```diff
@@ -0,0 +1,34 @@
+import json
+from typing import List, Tuple
+
+class KnownNames(object):
+    def __init__(self):
+        self.people = {}
+
+    def add_name(self, person: str, name: str):
```

Comment:
ATM, I've been blocked on mypy because the parser doesn't support format string literals yet - I was planning on adding that last weekend, but fires happened. Really, I should have it everywhere!